libraries/AP_Follow/AP_Follow.h

```diff
@@ -28,6 +28,11 @@ class AP_Follow
 
 public:
 
+    // enum for FOLLOW_OPTIONS parameter
+    enum FollowOptions {
```

Comment:
```suggestion
    enum class Option {
```

singular and move redundant namespacing